forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/main/java/com/forgerock/openbanking/aspsp/rs/api/payment/v3_1_3/domesticpayments/DomesticPaymentConsentsApiController.java

```diff
@@ -57,13 +58,28 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DomesticPaymentConsentsApiController.class);
 
+    /**
+     * Constructor
+     * @param rsEndpointWrapperService
+     * @param rsStoreGateway
+     * @param paymentsService
+     * @param objectMapper
+     */
```

Comment:
Minor - perhaps get rid of this javadoc (or complete the `@param` descriptions).

Reply:
Removed